src/codegen/codegen_ispc_visitor.cpp

```diff
@@ -292,6 +305,23 @@ std::string CodegenIspcVisitor::ptr_type_qualifier() {
     }
 }
 
+std::string CodegenIspcVisitor::global_var_struct_type_decorator() {
+    if (wrapper_codegen) {
+        return CodegenCVisitor::global_var_struct_type_decorator();
+    } else {
+        return "uniform ";  // @note: extra space needed to separate qualifier from var name.
```

Comment:
put the space not here but actually where you use the qualifier

Reply:
I am following the standard set by `ptr_type_qualifier()`